actions/UserLogin.ts

```diff
@@ -23,7 +23,7 @@ export class Action extends ActionBase<Entities.User> {
     }
 
     protected async execute(context): Promise<Entities.User> {
-        let userFromDb = (await this._userRepository.find({ username: context.params.username }));
+        let userFromDb = (await this._userRepository.findOneByQuery({ username: context.params.username }));
```

Comment:
@mrtomic I think we should name this findOne instead of findOneByQuery, it will be more consistent with mongo interface.